docs/source/openstack/volume-migration-using-s3.md

```diff
@@ -0,0 +1,190 @@
++-----------+-------------------------------------------------------+
+| endpoints | moc-kzn                                               |
+|           |   admin: https://kzn-swift.massopen.cloud/swift/v1    |
+|           | moc-kzn                                               |
+|           |   public: https://kzn-swift.massopen.cloud/swift/v1   |
+|           | moc-kzn                                               |
+|           |   internal: https://kzn-swift.massopen.cloud/swift/v1 |
+|           |                                                       |
++-----------+-------------------------------------------------------+
+```
+
+And from the NERC:
+
+```
+$ openstack --os-cloud nerc catalog show object-store -c endpoints
++-----------+----------------------------------------------------------------------------------------+
+| Field     | Value                                                                                  |
++-----------+----------------------------------------------------------------------------------------+
+| endpoints | regionOne                                                                              |
+|           |   admin: http://10.255.2.253:8080                                                      |
+|           | regionOne                                                                              |
+|           |   internal: http://10.255.2.253:8080/v1/AUTH_01234567890123456789012345689012          |
+|           | regionOne                                                                              |
+|           |   public: https://stack.nerc.mghpcc.org:13808/v1/AUTH_01234567890123456789012345689012 |
+|           |                                                                                        |
++-----------+----------------------------------------------------------------------------------------+
+```
+
+## Configure minio client
+
+For each environment, create a MinIO alias configuration using the base URL of the "public" interface of the object-store service and the access key and secret your generated earlier:
+
+```
+$ mc alias set kaizen https://kzn-swift.massopen.cloud 1234567890ABCDEF1234567890ABCDEF 1234567890ABCDEF1234567890ABCDEF
+$ mc alias set nerc https://stack.nerc.mghpcc.org:13808 FEDCBA0987654321FEDCBA0987654321 FEDCBA0987654321FEDCBA0987654321
+```
+
+## Create volume backup in source environment
+
+Create a volume backup in kaizen:
+
+```
+$ openstack --os-cloud kaizen volume backup create --container <bucket_name> <volume id or name>
+```
+
+In the above command line, `<bucket_name>` is the name of the object storage container that will hold your backup. This must be unique across all users, so using your username as a prefix is probably a good idea (e.g., `--container larsks-backups`).
+
+This will create a set of objects in the named bucket. You can use the MinIO client to inspect it:
+
+```
+$ mcli ls kaizen
```

Comment:
Error
```
~/Downloads/mcli ls kaizen
bash: /c/Users/joach/Downloads/mcli: No such file or directory
```
When I try to use mc instead of mcli it returns nothing.

Reply:
It's not returning nothing; you're getting an error that suggests you have either mis-installed the command or it has incorrect permissions.

Reply:
if I create `~/Downloads/mc mb kaizen/jweyl-backups` then when I run it I get
```
~/Downloads/mc ls kaizen
[2022-04-15 15:54:13 EDT]     0B jweyl-backups/
```
but when I run `~/Downloads/mc ls kaizen/jweyl-backups/` I get no responce